### ISSUE TITLE
Declare MSRV?

### ISSUE THREAD

**Issue by 8573** [sfackler/cargo-tree#104]
Trying to run `cargo tree`, I get this error message:

```
error: Found argument '--filter-platform' which wasn't expected, or isn't valid in this context
USAGE:
    cargo metadata --format-version <VERSION>
For more information try --help
```

This is when trying to run `cargo tree` with Rust 1.34.2 and its corresponding Cargo (Cargo 1.34.0), for minimum supported Rust version (MSRV) reasons. With current stable Rust and Cargo (Rust 1.42.0, Cargo 1.42.0), `cargo tree` works, because of which I suppose that `cargo tree` has an MSRV greater than 1.34.2 — maybe 1.41, by #86? I would like to request that `cargo tree`'s MSRV be documented.

**Comment by sfackler** [comment]
Yep, 1.41 is the current MSRV. The source of truth is what the CI build runs: https://github.com/sfackler/cargo-tree/blob/master/.circleci/config.yml#L5
